.circleci/config.yml

```diff
@@ -6,7 +6,9 @@ jobs:
     resource_class: xlarge
 
     docker:
-    - image: circleci/golang:1.12
+    - image: circleci/golang:1.13
```

Comment:
One of k8s.io/client-go dependencies is not building anymore because it is using `go1.12`. We might need to create parallel CI workflows to test with both versions...

Reply:
Fixed with workflows. The main package (`ddtrace`) is built and tested with `go1.12`